apps/to-do-api/host.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "2.0",
+  "logging": {
+    "logLevel": {
+      "default": "Information"
+    }
+  },
+  "extensions": {
+    "http": {
+      "routePrefix": ""
+    }
+  },
```

Comment:
use the default one if you don't have a specific need (less things to config/remember)